tools/util/include/cutlass/util/reference/host/tensor_fill.h

```diff
@@ -205,23 +211,22 @@ struct RandomGaussianFunc<complex<Element> > {
 
     Element reals[2];
 
-    for (int i = 0; i < 2; ++i) {
-      // Box-Muller transform to generate random numbers with Normal distribution
-      double u1 = double(std::rand()) / double(RAND_MAX);
-      double u2 = double(std::rand()) / double(RAND_MAX);
-
-      // Compute Gaussian random value
-      double rnd = std::sqrt(-2 * std::log(u1)) * std::cos(2 * pi * u2);
-      rnd = mean + stddev * rnd;
-
-      if (int_scale >= 0) {
-        rnd = double(int(rnd * double(1 << int_scale)));
-        reals[i] = from_real<Element>(rnd / double(1 << int_scale));
+    double rnd1 = detail::BoxMullerFunc(mean, stddev);
```

Comment:
they are both using `cos` now.  do we want to call `cos` and `sin` by turns?

Reply:
Yes, we're wasting some efficiency here, but I wasn't able to come up with an easy and comprehensible inline function (without a loop) which returns a dynamic number of random values. A fully efficient function would reuse the generated uniform values `u1, u2`, but this would make the function much less readable...

Reply:
i think we only need to change the complex and quantitation ones.  we can leave the real one untouched.  the helper function just need to return a pair of Gaussian random values by calling both cos and sin.